docs/local-backend-roles/localroles.md

```diff
@@ -0,0 +1,29 @@
+# Setting roles in your local backend
+
+Please make sure that you have setup the backend locally. Watch these videos if you have not done that: 
+- <a href="https://www.youtube.com/watch?v=haqPaPRrhPU">Video 1</a>
+- <a href="https://www.youtube.com/watch?v=2Ja5hH2nH1o&feature=youtu.be">Video 2</a>

```

Comment:
NIT: We can replace "Video 1" and "Video 2" by their respective YouTube video titles

Reply:
will do that